src/box/lua/schema.lua

```diff
@@ -424,7 +426,7 @@ box.begin = function(options)
         assert(builtin.box_txn_set_timeout(timeout) == 0)
     end
     if txn_isolation and
-       builtin.box_txn_set_isolation(txn_isolation) ~= 0 then
+       internal.txn_set_isolation(txn_isolation) ~= 0 then
```

Comment:
Nit: `builtin.box_txn_set_isolation` is now never used so its declaration should be removed from `ffi.cdef`.

Also, I don't quite like that `box.begin` will break a JIT trace in case any isolation level is specified, even the one that never yields. OTOH it shouldn't happen if the default isolation level is used so it should probably be acceptable.

Reply:
> Nit: builtin.box_txn_set_isolation is now never used so its declaration should be removed from ffi.cdef.

Fixed, thanks!
```diff
diff --git a/src/box/lua/schema.lua b/src/box/lua/schema.lua
index 6b8ecc648..d32d9257f 100644
--- a/src/box/lua/schema.lua
+++ b/src/box/lua/schema.lua
@@ -74,8 +74,6 @@ ffi.cdef[[
     box_txn_begin();
     int
     box_txn_set_timeout(double timeout);
-    int
-    box_txn_set_isolation(uint32_t level);
     void
     memtx_tx_story_gc_step();
     /** \endcond public */
```